server/datamodel.prisma

```diff
@@ -1,4 +1,28 @@
 type User {
   id: ID! @id
   name: String!
+  password: String!
+  email: String!
+  phone: Int
+  emergency: Int
+  booked: [Booking]!
+  favourite: [String]
 }
+type Campsite {
+  id: ID! @id
+  campname: String!
+  price: Int!
+  description: String
+  facilities: [String]
+  details: [String]
+  reviews: [String]
```

Comment:
Review should be a separate node with id, comment and who wrote it for which campsite.